src/features/comments/comments.ts

```diff
@@ -0,0 +1,80 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import {
+  createComment,
+  deleteComment,
+  getPostComments,
+} from '../../api/comments';
+import { Comment } from '../../types/Comment';
+
+export const fetchPostComments = createAsyncThunk(
+  'fetch/comments',
+  (postId: number) => getPostComments(postId),
+);
+
+export const addPostComment = createAsyncThunk(
+  'post/comment',
+  (data: Omit<Comment, 'id'>) => createComment(data),
+);
+
+export const deletePostComment = createAsyncThunk(
+  'delete/comment',
+  (commentId: number) => {
+    deleteComment(commentId);
+
+    return commentId;
+  },
+);
+
+type InitialState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: InitialState = {
+  comments: [],
+  loaded: true,
```

Comment:
The initial state for `loaded` should be `false` to correctly represent that data loading has not started. This will prevent any UI components from mistakenly assuming data is ready before any fetch operation.